osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -32,6 +33,8 @@ public class Slider : OsuHitObject, IHasCurve
 
         public Vector2 StackedPositionAt(double t) => StackedPosition + this.CurvePositionAt(t);
 
+        public Action OnRegenerated;
```

Comment:
documentation. also why is this called in nested object creation? shouldn't it be called `NestedGenerationComplete` or something..?